code/modules/modular_computers/file_system/programs/generic/ntsl2_interpreter.dm

```diff
@@ -3,6 +3,7 @@
 	filedesc = "NTSL2+ Interpreter"
 	extended_desc = "This program is used to run NTSL2+ programs."
 	program_icon_state = "generic"
+	usage_flags = PROGRAM_CONSOLE | PROGRAM_LAPTOP | PROGRAM_TABLET | PROGRAM_STATIONBOUND | PROGRAM_WRISTBOUND
```

Comment:
Why are you adding restrictions? Also this isn't mentioned in changelog. And why pAIs are excluded?